test/e2e/conformance.yaml

```diff
@@ -1,6 +1,9 @@
+podSpec:
+  restartPolicy: Never
+  serviceAccountName: sonobuoy-serviceaccount
 sonobuoy-config:
   driver: Job
-  plugin-name: azure-arc-ci-conformance
+  plugin-name: azure-arc-monitor
```

Comment:
@wanlonghenry , what's the reason for this name change?

Reply:
arc validation team reached to use for this change